app/helpers/merge_requests_helper.rb

```diff
@@ -20,6 +20,15 @@ def merge_request_pending_since mr
     "pending for #{time}"
   end
 
+  def merge_request_counter (value)
```

Comment:
Use just `merge_request_counter value`, project code style :-), in doubt run rubocop on it.